File: app/src/main/java/com/pkmnapps/activitydo/dataclasses/Widget.java
package com.pkmnapps.activitydo.dataclasses;

public class Widget {
    final int type;
    final Object object;
    int sortOrder;
    String uid;
    public Widget(int type, Object object, String uid, int sortOrder) {
        this.type = type;
        this.object = object;
        this.uid = uid;
        this.sortOrder = sortOrder;
    }

    public int getType() {
        return type;
    }

    public Object getObject() {
        return object;
    }

    public int getSortOrder() {
        return sortOrder;
    }

    public void setSortOrder(int sortOrder) {
        this.sortOrder = sortOrder;
    }

    public String getUid() {
        return uid;
    }

    public void setUid(String uid) {
        this.uid = uid;
    }
}

File: app/src/main/java/com/pkmnapps/activitydo/dataclasses/Widget.kt
package com.pkmnapps.activitydo.dataclasses

import org.junit.runner.RunWith

class Widget(val type: Int, val `object`: Any?, var uid: String?, var sortOrder: Int) {
    fun getType(): Int {
        return type
    }

    fun getObject(): Any? {
        return `object`
    }

    fun getSortOrder(): Int {
        return sortOrder
    }

    fun setSortOrder(sortOrder: Int) {
        this.sortOrder = sortOrder
    }

    fun getUid(): String? {
        return uid
    }

    fun setUid(uid: String?) {
        this.uid = uid
    }
}